src/coreclr/scripts/superpmi_benchmarks.py

```diff
@@ -187,12 +187,13 @@ def build_and_run(coreclr_args, output_mch_name):
         env_copy["NUGET_EXPERIMENTAL_CHAIN_BUILD_RETRY_POLICY"] = "9,2000"
 
     # If `dotnet restore` fails, retry.
+    tfm = "net9.0"
```

Comment:
Another option is to turn it into `superpmi_benchmarks.py.template` with markers like `{TargetFramework}` which some `.proj` would replace at build time to generate the real `superpmi_benchmarks.py` somewhere under `artificats/obj`. linker testing uses that mechanism to avoid updating the files each time the TFM version is bumped:

https://github.com/dotnet/runtime/blob/b85f9f621bf19ad12eb91a7b6bf35a1503c58845/eng/testing/linker/trimmingTests.targets#L94
template:
https://github.com/dotnet/runtime/blob/b85f9f621bf19ad12eb91a7b6bf35a1503c58845/eng/testing/linker/project.csproj.template#L1

Reply:
That's a good idea, but a bit more work than I'd like to take on here -- I opened #110043 about it